lib/account.rb

```diff
@@ -0,0 +1,161 @@
+require 'CSV'
+
+module Bank
+
+  class Account
+
+    attr_accessor :id, :balance, :created
```

Comment:
Should these be `attr_accessor`? Seems to me that `attr_reader` would be more appropriate - you don't want people trying to modify `@balance` directly, they should go through `withdraw` or `deposit` instead.